recipes/emscripten/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.0.8" %}
+
+package:
+  name: emscripten
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/emscripten-core/emscripten/archive/{{ version }}.tar.gz
+  sha256: 609a7ecea954821efbcc55bf5ad88dd03ba4f836c703b8ad2e37c243329174a6
+  patches:
+    - remove_revision_file_setting.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - binaryen
+    - nodejs
+  run:
+    - python
```

Comment:
```suggestion
    - python *
```

Reply:
I don't think that'd work since conda-build compiles the python scripts to bytecode and ships that.

Reply:
That can be disabled: https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skip-compiling-some-py-files-into-pyc-files

Reply:
yeah, not sure. This is, and is not a Python package... I don't really know how to handle this.
I think it's correct to build out the package for multiple python versions, so that the pyc files are also correctly erased when removing the package.